controller.go

```diff
@@ -94,6 +94,9 @@ type NetworkController interface {
 	// Sandboxes returns the list of Sandbox(s) managed by this controller.
 	Sandboxes() []Sandbox
 
+	// RenameSandbox changes the sandbox name; should be called on a container rename
+	RenameSandbox(containerID string, options ...SandboxOption) error
```

Comment:
Could we have a `Rename()` method in `Sandbox` interface instead ?
Docker already holds the SandboxID for each container, it would be a direct lookup by calling `sb,err  :=SandboxByID()` and then a `sb.Rename()`